platform-api/src/test/java/io/datacater/core/deployment/DatacaterDeploymentEndpointTest.java

```diff
@@ -124,12 +124,36 @@ void testGetDeployment() {
 
   @Test
   @Order(3)
+  void testGetDeploymentStatusByValidUuid() {
+    given()
+        .pathParam("uuid", deploymentId)
+        .baseUri(baseURI)
+        .get(deploymentsPath + "/{uuid}/status")
+        .then()
+        .statusCode(200);
+  }
```

Comment:
The reason this is failing is because your `[DeploymentEndpoint.getDeploymentStatus()](https://github.com/DataCater/datacater/blob/e109b00e7b5971ae74ac3e99458795231db097d3/platform-api/src/main/java/io/datacater/core/deployment/DeploymentEndpoint.java#L476)` method is returning null on getStatus(). I beieve this is the case when the deployment is still starting and not yet in  a healthy state. i would suggest using the already implemented [getDeployment()/deploymentToMetaDataMap()](https://github.com/DataCater/datacater/blob/main/platform-api/src/main/java/io/datacater/core/deployment/K8Deployment.java#L216-L257) methods, these will return the status if it exists and also some other metadata info, so the user actually gets the information that the deploymet is still starting. Doing that should fix this test